components/calculator/RecoltoCalculator.vue

```diff
@@ -119,15 +121,15 @@ const emit = defineEmits([
 const steps = [
   {
     value: 1,
-    label: "Toiture",
+    label: t("calculateur.roof"),
```

Comment:
```suggestion
    label: t("calculator.roof"),
```

and everywhere else ;)